core/storage/user/gae_models.py

```diff
@@ -148,6 +148,20 @@ class UserStatsModel(base_models.BaseMapReduceBatchResultsModel):
     # The impact score.
     impact_score = ndb.FloatProperty(indexed=True)
 
+class UserDashboardStatsModel(base_models.BaseMapReduceBatchResultsModel):
```

Comment:
@seanlip I did follow your suggestion and reused the existing model instead of creating a new one. But the [instrcutions here](https://github.com/oppia/oppia/wiki/Calculating-statistics#finding-the-right-job) clearly state that no 2 jobs should be updating the same model.

Thus I'm forced to think that we might have to:
1. either revert back to using different models,
2. or re-write the existing impact aggregator job to be able to make the necessary aggregations for average ratings and number of plays as well.
I'm not sure if  (2) is even feasible -- your take on this ?